lib/renderer/web-view/web-view.js

```diff
@@ -341,7 +341,11 @@ const registerWebViewElement = function () {
     if (!internal.guestInstanceId) {
       internal.createGuestSync()
     }
-    return remote.getGuestWebContents(internal.guestInstanceId)
```

Comment:
Can you make `if (!remote)` check at the very start of the function?

Reply:
done